services/ui-src/src/types/banners.ts

```diff
@@ -0,0 +1,23 @@
+// BANNER
+
+export interface BannerData {
+  title: string;
+  description: string;
+  link?: string;
+  [key: string]: any;
+}
+
+export interface AdminBannerData extends BannerData {
+  key: string;
+  startDate: number;
+  endDate: number;
+  isActive?: boolean;
+}
```

Comment:
Naturally, if we change the type def in the API, we should change it here to match. I think this will match my previous suggestion there?

```suggestion
export interface BannerData {
  title: string;
  description: string;
  link?: string;
  key: string;
  startDate: number;
  endDate: number;
  isActive?: boolean;
}
```